src/time-utils.js

```diff
@@ -0,0 +1,39 @@
+'use strict'
+const ms = require('ms')
+
+function staleDaysToDate(input) {
+  try {
+    const staleDays = Number(input)
+    if (isNaN(staleDays)) {
+      const stringToMs = ms(input)
+      return new Date().getTime() - stringToMs
+    }
+    return new Date().getTime() - daysToMs(staleDays)
+  } catch (error) {
+    return new Date().getTime()
+  }
+}
+
+function isCommitStale(unreleasedCommits, staleDate) {
+  if (!unreleasedCommits || !unreleasedCommits.length) return false
+  return unreleasedCommits.some((commit) => {
+    const commitDate = new Date(commit.commit.committer.date).getTime()
+    return commitDate < staleDate
+  })
+}
+
+function isClosedNotifyIssueStale(closedNotifyIssues, staleDate) {
```

Comment:
this should accept a single issue and it doesn't care if it's a "closed notify issue". it can be any issue really. reflect this in the naming